torch/nn/quantized/modules/conv.py

```diff
@@ -103,6 +103,42 @@ def forward(self, input):
                                              self.scale, self.zero_point)
         return output.permute([0, 3, 1, 2])
 
+    # ===== Serialization methods =====
+    # The special consideration here is that we have to unpack the weights into their
+    # regular QTensor form for serialization. Packed weights should not live
+    # outside the process in which they were created, rather they should be derived
+    # from the QTensor weight.
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(Conv2d, self)._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'weight'] = self.weight()
+        destination[prefix + 'scale'] = torch.tensor(self.scale)
+        destination[prefix + 'zero_point'] = torch.tensor(self.zero_point)
+        if self.bias is not None:
+            destination[prefix + 'bias'] = self.bias
+
+    # ===== Deserialization methods =====
+    # Counterpart to the serialization methods, we must pack the serialized QTensor
+    # weight into its packed format for use by the FBGEMM ops.
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        self.set_weight(state_dict[prefix + 'weight'])
+        state_dict.pop(prefix + 'weight')
+
+        if prefix + 'bias' in state_dict:
+            self.bias.copy_(state_dict[prefix + 'bias'])
+            state_dict.pop(prefix + 'bias')
+        else:
+            self.bias = None
```

Comment:
Do we need special handling of self.bias in get_state and set_state functions?

Reply:
Actually, I'm going to remove the special handing in these functions since I just discovered it's not needed